eg-next/README.md

```diff
@@ -76,3 +76,8 @@
 
 [![A screenshot of the embedded-graphics version next example called text-transparent](../doc/assets/next/text-transparent.png)](examples/text-transparent.rs)
 
+
+## [textbox-alignment](examples/textbox-alignment.rs)
```

Comment:
I'm not sure about the name of the demo. I would like to use the `text-` prefix, because it is a text related feature and contain `embedded-text`, because it is a demo for that external create. But `text-embedded-text` doesn't sound great either.

Reply:
`text-basic-layout` is the best I can come up with

Reply:
I was considering merging the example into `text-multiline` or `text-alignment` and I didn't like either option. Truth is, this example is a bit of both.

I'm afraid that users would miss the fact the example uses an "external" crate if we keep the names so similar.

Reply:
Maybe `text-textbox`?

In the future we could update the `README.md` to include sections and maybe some description per example. But for now we should just pick any of the suggested names. This can easily be improved later without breaking anything.

Reply:
`text-textbox` is good enough. Since there exists a single example right now, I've used this name. Later this might turn into a prefix.